chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,6 +150,12 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
+            # we want disks that are multipath but not if they are holders
+            # of a multipath
+            if d.multipath?
```

Comment:
Why use `next` at all?  Isn't this equivalent?

```ruby
!(d.claimed? || d.held_by_multipath?) &&
d.fixed || d.multipath?
```

or this?

```ruby
next unless d.fixed || d.multipath?
!(d.claimed? || d.held_by_multipath?)
```

I'm not saying they are better, I'm just trying to understand why you're suggesting that particular form.  It seems you're thinking about them as two pairs of two expressions, and maybe there's a good reason for doing that (as suggested by this `should_be_included` idea) but it's not clear to me what that reason is.